src/internal/docker/cli.go

```diff
@@ -184,25 +189,39 @@ func startContainer(ctx context.Context, containerName string, options RunOption
 	return run(ctx, "start", "-i", containerName)
 }
 
-func execContainer(ctx context.Context, containerName string, cmd []string, options RunOptions) error {
+func execContainer(ctx context.Context, yeyCtx yey.Context, containerName string, options RunOptions) error {
```

Comment:
I think, ideally, the docker package should work without knowing about yey.Context. It wouldn't be a crime, but it would be nice!

Reply:
meh it's fine.

Reply:
ça me dérange pas que le package docker soit couplé au `yey.Context` parce que toutes les informations dont on a besoin pour exécuter nos commandes docker cli sont dans cet objet et autrement, faudrait tout répliquer dans un autre objet équivalent, juste "parce que".  Et à chaque fois qu'on rajoute une nouvelle propriété, ça complique tout. Présentement, c'est très simple et très maintenable. Comme je disais, si on voulait exposer ce package pour qu'il soit réutilisable à l'extérieur du projet, ça serait une autre histoire.

Reply:
it's the yey-docker package. So that's why I changed it to "meh nevermind"